8wtm_auto_switch

```diff
@@ -78,9 +80,17 @@ if [[ -e "$WTM_HISTORY" ]] ; then
   echo "$(cat $WTM_HISTORY | tail -n 100)" > $WTM_HISTORY
 fi
 
-# Check last top coin and change if not in coin list any more
 FIRST_COIN="$(echo "$WTM_AUTO_SWITCH_COINS" | awk -F'[_;]' '{print $1}')"
+
+# Prevent first run error on no top coin
+if [[ ! -e ${NVOC}/WTM_top_coin ]]
+then
+  echo "$FIRST_COIN:100" > ${NVOC}/WTM_top_coin
+fi
+
 TOP_COIN=$(sed 's/:[0-9]*//' < ${NVOC}/WTM_top_coin)
+
```

Comment:
Avoid immediate read after write by setting TOP_COIN=$FIRST_COIN in the above if block and move the read from file command at line 91 in its else branch.

Reply:
Better now?

Reply:
Have you pushed the new commit?

Reply:
Yup, but not showing up 
No idea why

Reply:
Showing in branch, but not in PR
https://github.com/papampi/nvOC_by_fullzero_Community_Release/blob/papampi-WTM_top_coin-patch/8wtm_auto_switch

Reply:
I see, seems that server are overloaded a.t.m.